pkg/webui/styles/utilities/general.styl

```
@@ -97,6 +97,15 @@
       .{$name}:j-between
         justify-content: space-between
 
+      .{$name}\\:al-center
```

```suggestion
      .{$name}:al-center
```
The slash escapes are actually not required.